Public/Src/Engine/UnitTests/Processes.Detours/PipExecutorDetoursTest.cs

```diff
@@ -6163,6 +6163,46 @@ private static FileArtifact WriteFile(PathTable pathTable, AbsolutePath filePath
             return FileArtifact.CreateSourceFile(AbsolutePath.Create(pathTable, expandedPath));
         }
 
+        [Fact]
+        public async Task CallDeleteWithoutSharing()

```

Comment:
>CallDeleteWithoutSharing [](start = 26, length = 24)

nit - could you add some comments or tweak the name to make it more clear what you're testing? UntrackedScopesDoNotGetFileShareDeleteAdded()?

Reply:
Added unit test to cover the tracked path.